src/Mvc/Mvc.Core/src/ModelBinding/Binders/DateTimeModelBinderProvider.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
+{
+    /// <summary>
+    /// An <see cref="IModelBinderProvider"/> for binding <see cref="DateTime" /> and nullable <see cref="DateTime"/> models.
+    /// </summary>
+    public class DateTimeModelBinderProvider : IModelBinderProvider
+    {
+        internal static readonly DateTimeStyles SupportedStyles = DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces;
```

Comment:
Should this be configurable?

Reply:
You can create a binder provider that configures the model binder (it's constructable). It's the pattern we use for other types.